### DISCUSSION TITLE
Pimcore Portal Engine for Pimcore X

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#9208] [Q&A / Support]
Does Pimcore portal engine also supports Pimcore X?
As per documenation it suports pimcore 6.9 

```
Following requirements additionally to Pimcore apply:

    PHP 7.4
    Pimcore 6.9
```

https://pimcore.com/docs/portal-engine/current/Installation/Installation.html

**Comment by fashxp** [comment]
yes, should be read as minimum requirement, will fix that.

**Reply by nirbanc2020** [reply]
I installed Pimcore Portal Engine in Pimcore X version using Php 8.03 . While installing I got the following error. Please check the screenshot.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/73533273/119670877-e72ec900-be56-11eb-95f1-ed668989bea0.png)

**Reply by jdreesen** [reply]
The problem is that its dependency `ongr/elasticsearch-dsl` does not support PHP 8 yet.

**Reply by fashxp** [reply]
please use --ignore-platform-req=php in composer statements in the meanwhile.

**Reply by solverat** [reply]
You may want to replace it with https://github.com/handcraftedinthealps/elasticsearchdsl. It‘s maintained by the guys of sulu so it‘s in good hands i guess. I‘m also going to use it for the px version of dynamic search.

**Reply by nirbanc2020** [reply]
@fashxp Thanks.... I used --ignore-platform-req=php in composer statements while installing Pimcore portal engine.It is working and got installed..

**Reply by fashxp** [reply]
> You may want to replace it with https://github.com/handcraftedinthealps/elasticsearchdsl. It‘s maintained by the guys of sulu so it‘s in good hands i guess. I‘m also going to use it for the px version of dynamic search.

thx for the hint

**Reply by fashxp** [reply]
seems there is some progress going on there: https://github.com/ongr-io/ElasticsearchDSL/issues/344#issuecomment-849444927 ... so lets see.

**Reply by solverat** [reply]
If I'm getting this right, this is going to be another fork I guess (@84m is not a member of @ongr-io). I've already contacted the main company (NFQ) by mail - so again: let's see. :)

**Reply by nirbanc2020** [reply]
> seems there is some progress going on there: [ongr-io/ElasticsearchDSL#344 (comment)](https://github.com/ongr-io/ElasticsearchDSL/issues/344#issuecomment-849444927) ... so lets see.

I uninstalled previous version elastic search and install elastic search from https://github.com/handcraftedinthealps/elasticsearchdsl.But in that case asset pool is not working properly. I think it is not properly support php-8.

**Reply by solverat** [reply]
There are other bundles using this fork within the php8 ecosystem with no (known) issues. There might be some issues on pimcore side - but I haven't tested it.

**Reply by fashxp** [reply]
using ongr-io with `ignore-platform-req=php` works just fine for us. don't have any experience with other forks.

**Reply by nirbanc2020** [reply]
I installed Portal engine in server.After creating virtual host and configuring the portal engine ..It is not opening in new domain. The error comming "This page isn’t working too may redirects" .Can anyone help me regarding the same? In local machine it is running properly.
